delta/plugins/storage/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/storage/storages/operations/s3/client/S3StorageClientImpl.scala

```diff
@@ -76,6 +70,56 @@ final private[client] class S3StorageClientImpl(client: S3AsyncClientOp[IO], val
         .build()
     )
 
+  def copyObjectMultiPart(
+      sourceBucket: String,
+      sourceKey: String,
+      destinationBucket: String,
+      destinationKey: String
+  ): IO[CompleteMultipartUploadResponse] = {
+    val partSize = 5L * 1024L * 1024L // 5MB
```

Comment:
That may be a little low no ?

Reply:
Also some files are empty, we have to make sure it works, maybe for files lower than the part size we should not use multipart ?